groups/chips-db/locals.tf

```diff
@@ -30,8 +30,12 @@ locals {
 
   internal_fqdn = format("%s.%s.aws.internal", split("-", var.aws_account)[1], split("-", var.aws_account)[0])
 
-  oracle_allowed_ranges = concat(local.internal_cidrs, var.vpc_sg_cidr_blocks_oracle)
```

Comment:
You should be able to get rid of the local internal_cidrs var and corresponding vault data resource as it is no longer used

Reply:
@JMetherell Good spot. I realised I had un-pushed commits so this is now resolved.